res/css/structures/_AutoHideScrollbar.pcss

```diff
@@ -1,16 +1,19 @@
 /*
 Copyright 2018-2024 New Vector Ltd.
+Copyright 2024 Suguru Hirahara
 
 SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only
 Please see LICENSE files in the repository root for full details.
 */
 
-/* make any scrollbar grey and thin */
+/* make any scrollbar grey */
 html {
     scrollbar-color: $scrollbar-thumb-color transparent;
 }
+/* SC: disabled */
```

Comment:
I would put it right above the actual commented out section, and use the same `/*`. The first point further reduces merge conflict area, the second one is just matter of taste I guess